### PR TITLE
INT-173 : Add support for /employees[/:id] endpoints

### PR DESCRIPTION
**Pull request by muhammadatdoshii**
[INT-173](https://doshii.atlassian.net/browse/INT-173)

### CODE REVIEW

**Comment by johnatdoshii** [comment]
@muhammadatdoshii can you please ensure your commit messages include the Jira ticket number so they can be correctly linked to the work. e.g. this commit should be:

```
INT-173 : add support for getting employee(s)
```

When you Squash & Merge after this PR is approved, can you please adjust the commit message?

**Comment by muhammadatdoshii** [comment]
> @muhammadatdoshii can you please ensure your commit messages include the Jira ticket number so they can be correctly linked to the work. e.g. this commit should be:
> 
> ```
> INT-173 : add support for getting employee(s)
> ```
> 
> When you Squash & Merge after this PR is approved, can you please adjust the commit message?

Understood. Will do.

**Review by johnatdoshii — CHANGES_REQUESTED** [review]
Looks good - can we also update this PR to include a change to `package.json` to increment the version to `4.0.9`?

Happy to approve once that file change is in. 💪

**Comment by muhammadatdoshii** [comment]
> Looks good - can we also update this PR to include a change to `package.json` to increment the version to `4.0.9`?
> 
> Happy to approve once that file change is in. 💪

Roger roger.

**Comment by johnatdoshii** [comment]
@carlosatdoshii FYI, @muhammadatdoshii has added the employee endpoints to the SDK. 🎉

**Review by johnatdoshii — APPROVED** [review]

File: lib/employee.ts
Comment (muhammadatdoshii) [reply lib/employee.ts:19]: Yeah, that's a good point. Although having all three does not necessarily mean they are redundant - it's merely exposing a range of intuitive methods one would expect from the SDK. And making it private doesn't add much value in terms of abstracting away details.

Comment (johnatdoshii) [reply lib/employee.ts:19]: It's certainly a good point. I agree with Muhammad in that it doesn't necessarily make the other functions redundant, it just makes it less obvious as to which function should be called.

Making it private does make it cleaner, but does it remove some flexibility that is a nice to have? I'm not sure of the answer, to be honest. I'll be guided by the 3 of you. I'm happy to make the change if we think this improve things - my only concern is that if we do remove it (and across other classes), it introduces a breaking change, so we'd need to move to `5.0.0` as other partners might be using them. :thinking:

Comment (carlosatdoshii) [reply lib/employee.ts:19]: Ok, I really think `getAll` and `getOne` are redundant, mainly because the get method public accessible and has this flexibility by itself, however it's not a big deal for me.

Having said that, if you guys think that create a major version could be an overhead, then Im happy to keep as it is. But the only think I see as a **must** is JSDoc needed for all public methods for an SDK, so please let's add it to this get method in here.

Comment (muhammadatdoshii) [reply lib/employee.ts:19]: @johnatdoshii I don't think it should be a very massive undertaking. I'll have a look and create a ticket

**Pull request merged**
